tomcat-main/src/main/resources/tenants/anthro/anthro-collectionobject.xml

```diff
@@ -5,6 +5,7 @@
 		<repeat id="objectNameList/objectNameGroup">
 			<!-- Add mini -->
 			<field id="objectName" mini="list" />
+			<field id="objectNameControlled" mini="list" autocomplete="true" />
```

Comment:
When tenant overlays are merged into the base config, attributes in the overlay are added to attributes in the base. Since `autocomplete="true"` is in the base file, you don't need it here. It's easier to discern what's being changed if you don't repeat the attributes that are inherited and not being changed. I also like to add a comment above every line that is being modified in the overlay, like the `<!-- Add mini -->` one above, to make it really clear what the purpose is.